packages/esm-active-visits-app/src/active-visits-widget/active-visits.component.tsx

```diff
@@ -83,56 +90,69 @@ const ActiveVisitsTable = (props) => {
         gender: visit?.patient?.person?.gender,
         age: visit?.patient?.person?.age,
         visitType: visit?.visitType.display,
+        patientUuid: visit?.patient?.uuid,
       }));
       setActiveVisits(rowData);
       setLoading(false);
     });
     return () => activeVisits.unsubscribe();
   }, []);
-  return (
+
+  return !loading ? (
     <div className={styles.activeVisitsContainer}>
       <div className={styles.activeVisitsDetailHeaderContainer}>
-        <h4 className={styles.productiveHeading02}>{t('activeVisits', 'Active Visits in Clinic')}</h4>
+        <h4 className={styles.productiveHeading02}>{t('activeVisits', 'Active Visits')}</h4>
```

Comment:
Changed here but not in the actual language file.

Reply:
Ohh yeah, Thanks!